resources/js/features/games/components/GameSidebarFullWidthButtons/SidebarContributeLinks/SidebarManagementSection/SidebarManagementSection.tsx

```diff
@@ -70,7 +70,8 @@ export const SidebarManagementSection: FC<SidebarManagementSectionProps> = ({ ga
         {can.manageGameHashes ? (
           <PlayableSidebarButton
             href={`/manage/games/${backingGame.id}/hashes`}
-            IconComponent={LuWrench}
+            target="_blank"
+            IconComponent={LuFileArchive}
```

Comment:
Shouldn't this use the same icon as the "Supported Game Files" item? (They could both use the new one).

Reply:
Swapped to `LuFileCheck` in latest.